cves/CVE-2015-3982.yml

```diff
@@ -171,10 +186,11 @@ interesting_commits:
     interesting in light of the lessons learned from this vulnerability. Any
     emerging themes?
   commits:
-  - commit:
-    note:
-  - commit:
-    note:
+  - commit: 33076af6f2aa5285b3a70246e14163b901b512f7
+    note: |
+      This was the first commit after the vcs was added to django. What did it address?
+      An issue which could allow attackers to manipulate session data using the cache. which is similar the vulnerablity that
```

Comment:
Should the period be a comma? As in "using the cache, which is similar to the vulnerability"